CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Major: Newly uploaded Twitch emotes are present in emote picker and can be "Tabbed" as well. (#2992)
```

Comment:
```suggestion
- Major: Newly uploaded Twitch emotes are once again present in emote picker and can be "Tabbed" as well. (#2992)
```
I'm just going to suggest this changelog variation purely because as far as most people are concerned, this was just a bug, and they don't know what had to be done to fix it.